cirq-core/cirq/ops/fsim_gate.py

```diff
@@ -202,20 +214,24 @@ def _json_dict_(self) -> Dict[str, Any]:
 
 @value.value_equality(approximate=True)
 class PhasedFSimGate(gate_features.InterchangeableQubitsGate, raw_types.Gate):
-    """General excitation-preserving two-qubit gate.
+    r"""General excitation-preserving two-qubit gate.
 
     The unitary matrix of PhasedFSimGate(θ, ζ, χ, γ, φ) is:
 
-        [[1,                       0,                       0,            0],
-         [0,    exp(-iγ - iζ) cos(θ), -i exp(-iγ + iχ) sin(θ),            0],
-         [0, -i exp(-iγ - iχ) sin(θ),    exp(-iγ + iζ) cos(θ),            0],
-         [0,                       0,                       0, exp(-2iγ-iφ)]].
+    $$
+    \begin{bmatrix}
+        1 & 0 & 0 & 0 \\
+        0 & e^{-i \gamma-i \zeta} & -i e^{-i \gamma + i\chi} & 0 \\
```

Comment:
nit: add spaces around the second minus in the first exponential for consistency with the others